### PR TITLE
Write some feed tests

### PR DESCRIPTION
**Pull request by lishaduck**
## Description

This PR writes some tests that @MattsAttack forgot to write in #6.

---

## Type of Change

- 🧪 Test

## Checklist

```[tasklist]
### Checklist
- [x] Read the Contributing Guidelines.
- [x] Check that there isn't already a PR that solves the problem the same way to avoid creating a duplicate.
- [x] Fill out this template.
- [ ] Log your hours.
- [x] Check that commits follow the Angular commit convention, more or less.
- [x] Ideally, include relevant tests that fail without this PR but pass with it (if applicable).
```

## Tested on

- macOS 14

### CODE REVIEW

**Comment by lishaduck** [comment]
A dummy force-push fixed the CI flake, I think this is good to merge once dependency PRs are merged.

**Comment by lishaduck** [comment]
The improvements to the router testing are all that's needed before I'll give a 👍 to this (and dependencies).

**Review by MattsAttack — APPROVED** [review]

**Pull request merged**
